doc/version_history.rst

```diff
@@ -6,6 +6,16 @@
 Version History
 ###############
 
+v1.2.0
+------
+
+* Adapt the simplified state machine and updated interface with the controller.
```

Comment:
You're using present tense (`adapt`) and passed tense (`updated`) in the same sentence. Please choose one and use that for all verbs.

Reply:
Fixed in the commit: [1d80bf8](https://github.com/lsst-ts/ts_mthexapod/pull/81/commits/1d80bf8d9e5add0ba14ef74ccfd3cce7efd80e76). Thanks!